content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -86,8 +86,8 @@ Because ClusterRoles are cluster-scoped, you can also use them to grant access t
 * cluster-scoped resources (like {{< glossary_tooltip text="nodes" term_id="node" >}})
 * non-resource endpoints (like `/healthz`)
 * namespaced resources (like Pods), across all namespaces
-  For example: you can use a ClusterRole to allow a particular user to run
-  `kubectl get pods --all-namespaces`.
+  (for example: you can use a ClusterRole to allow a particular user to run
+  `kubectl get pods --all-namespaces`)
```

Comment:
Another way to put this. Note there is an empty line inserted. The "For example: ..." line keeps the same indentation as the bullet items.

```
* namespaced resources (like Pods), across all namespaces

  For example: you can use a ClusterRole to allow a particular user to run
  `kubectl get pods --all-namespaces`.
```

Reply:
I agree with @tengqm on this one, @edwardrosen - would you mind updating such that there's a new indentation line instead of putting it in parenthesis?

Reply:
+1 on this as well.